Sources/LeafKit/LeafParser/LeafParser.swift

```diff
@@ -200,7 +200,13 @@ internal struct LeafParser {
                     let params = try readParameters()
                     // parameter tags not permitted to have bodies
                     if params.count > 1  { group.append(.expression(params)) }
-                    else { group.append(params.first!) }
+                    ///// else { group.append(params.first!) }
+                    else {
+                      guard let firstParam = params.first else {
+                        throw LeafError(.missingParameter("Expected a parameter but found none"))
```

Comment:
I would prefer that this message be returned from `LeafError.localizedDescription`, and that the new error case not have the associated string at all.
```suggestion
                        throw LeafError(.missingParameter)
```